instabug/manifest.json

```diff
@@ -0,0 +1,49 @@
+{
+  "manifest_version": "2.0.0",
+  "app_uuid": "9cdb1efc-bdb7-4977-84df-07daa8742850",
+  "app_id": "instabug",
+  "display_on_public_website": false,
+  "tile": {
+    "overview": "README.md#Overview",
+    "configuration": "README.md#Setup",
+    "support": "README.md#Support",
+    "changelog": "CHANGELOG.md",
+    "description": "Monitor and track your mobile app health and performance",
+    "title": "Instabug",
+    "media": [],
```

Comment:
We highly recommend including images like dashboards, graphs, etc

Reply:
Hi @AbdallahHemdan - saw you marked this as resolved. We highly recommend images be included in the offering. Are you able to include images here?